hassio/src/snapshots/hassio-snapshot.js

```diff
@@ -282,14 +278,29 @@ class HassioSnapshot extends PolymerElement {
       );
   }
 
-  _computeDownloadUrl(snapshotSlug) {
-    const password = encodeURIComponent(this.hass.connection.options.authToken);
-    return `/api/hassio/snapshots/${snapshotSlug}/download?api_password=${password}`;
-  }
-
-  _computeDownloadName(snapshot) {
-    const name = this._computeName(snapshot).replace(/[^a-z0-9]+/gi, "_");
-    return `Hass_io_${name}.tar`;
+  _downloadClicked() {
+    this.hass
+      .callWS({
```

Comment:
We should make this a helper function in `auth/data.ts` (like the other functions in there)